.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
```

Comment:
Why did this file change?

Reply:
Auto formatting changes it. Is that a significant change ?

Reply:
I changed it back

Reply:
I am just surprised that this file changed and wanted to know why.